src/sprites/EntitySprite.js

```diff
@@ -55,6 +57,11 @@ export default class extends Phaser.Sprite {
     });
 
     this.events.onDragUpdate.add((entitySprite, pointer, x, y) => {
+      if (pointer.button !== 0) {
+        entitySprite.input.stopDrag(pointer);
```

Comment:
I was trying to solve this also yesterday but could not get it working. I guess, this is the best way.